packages/azure-http-specs/specs/azure/core/page/mockapi.ts

```diff
@@ -65,7 +65,8 @@ Scenarios.Azure_Core_Page_withParameterizedNextLink = passOnSuccess([
       body: json({
         value: [{ id: 1, name: "User1" }],
         // Include the original query parameters in the nextLink
-        nextLink: "/azure/core/page/with-parameterized-next-link/second-page?select=name",
+        nextLink:
+          "http://localhost:3000/azure/core/page/with-parameterized-next-link/second-page?select=name",
```

Comment:
I see other tests use `${req.baseUrl}` for the host instead of a hard-coded `http://localhost:3000`

Reply:
yeah I think this might break other uses which use custom ports

Reply:
Agree. .NET would use different port.